### DISCUSSION TITLE
Two categories to one payment

### DISCUSSION THREAD

**Discussion by Markobros** [moneymanagerex/moneymanagerex#6341] [Ideas]
Hello.
Didn't you think about two categories to one payment?
I don't mean a split transaction but one transaction that belongs into two categories.
For example: I'm on a holiday and pay for entrance ticket to a castle. So the payment belongs to category "tickets" and at the same time to category "holiday".
Because I want to know how much I spent not only for whole holiday but also for all entrance tickets.
I hope I explained it clearly.

**Comment by rmelillo76** [comment]
Hi there, not exactly what you are requesting but version 1.7, (still in beta but close to release) introduces "tag" capabilities. You'll be able to assign one or more tags to each transaction and those transactions can be easily identified through transaction filter and reporting.

**Reply by luisdev** [reply]
That's right - Tags is the best thing to happen to MMX in a long time! :-)

So, in MMX 1.7 you will create a **Tag** called "Holiday2023". Then for those entrance tickets you use the "Tickets" category when you record the transaction. And you attach the "HolidayNov2023" TAG to that tickets transaction.

That then allows you to create a report that filters on the "HolidayNov2023" Tag. It will show you all the transactions, across all the different categories, linked to your holiday and give you a nice report of what the holiday cost you.

**Reply by Markobros** [reply]
Thanks to both of you, that sounds good. Is there any possibility to download and try this beta version?

**Reply by rmelillo76** [reply]
Hi @Markobros there sure is! Here's a direct link to the beta version download page:
https://ci.appveyor.com/project/whalley/moneymanagerex

And some info when it comes to beta releases and Linux/MacOS builds:
https://moneymanagerex.org/docs/downloads/development/

**Reply by Markobros** [reply]
Thanks a lot. I tried it and it looks good. It means I'll have to go through my entire transaction history and sometimes it'll be a bit confusing to decide what's category and what's tag, but I think it'll be fine.
